aikau/src/main/resources/alfresco/core/Core.js

```diff
@@ -408,11 +408,15 @@ define(["dojo/_base/declare",
             topics = [topics];
          }
 
+         /*jshint -W059 */
+         var callerName = arguments.callee.caller.name;
```

Comment:
I'm aware of this... when it finally stops being supported, we'll do something about it.